test/run_test.py

```diff
@@ -232,6 +243,9 @@ def test_distributed(executable, test_module, test_directory, options):
     'cuda_primary_ctx': test_cuda_primary_ctx,
     'cpp_extensions': test_cpp_extensions,
     'distributed': test_distributed,
+    'ge_config_simple' : test_jit_ge,
```

Comment:
I really feel instead of adding these configurations we should just have `--ge_config` switch that can be read in `common_utils.py`, so it works for **all** files that export it.
Then the invocation in `test.sh` simply becomes an extra line: `run_test.py --include jit --ge_config=legacy`

@suo @zdevito

Reply:
I'd prefer these be test target names. The ge_config flag is not easily discoverable, and adds an additional construct to our test infrastructure. Adding new test names just builds on our existing constructs.

Reply:
@zdevito , so you would like me to add 3 new files:

* test_jit_simple.py
* test_jit_legacy.py
* test_jit_fuser_legacy.py

and the guts of each file will look something like this: 

test_jit_simple.py
```
GRAPH_EXECUTOR = ProfilingMode.EXECUTOR # for test_jit_simple.py
from test_jit import *

if __name__ == '__main__':
    run_tests()
    if not PY2:
        import test_jit_py3
        suite = unittest.findTestCases(test_jit_py3)
        unittest.TextTestRunner().run(suite)
```